doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
@venkuppu-chn, change file path from CORTX_on_Open_Virtual_Appliance_PI-5.rst to CORTX_on_Open_Virtual_Appliance_PI-6.rst

Reply:
Done. Thanks Mukul.